peps/management/commands/generate_pep_pages.py

```diff
@@ -129,3 +110,30 @@ def verbose(msg):
                     verbose("- Skipping non-PEP related image '{}'".format(img))
 
         verbose("== Finished")
+
+    def get_artifacts_path(self, stack, verbose):
+        if settings.DEBUG and settings.PEP_REPO_PATH:
```

Comment:
I don't think we should go back to the old method. Downloading and extracting an artifact from S3 sounds like a better idea to me.

I think we can set `settings.PEP_ARTIFACT_URL` to `os.path.join(settings.BASE, 'peps/tests/peps.tar.gz')` in `settings/local.py`. This way contributors wouldn't need to close python/peps if they would like to work on our PEP converter code.

Reply:
👌